src/Stratis.Bitcoin.Features.RPC/ModelBinders/DestinationModelBinder.cs

```diff
@@ -19,14 +19,9 @@ public Task BindModelAsync(ModelBindingContext bindingContext)
                 return Task.CompletedTask;
             }
 
-            ValueProviderResult val = bindingContext.ValueProvider.GetValue(
-                bindingContext.ModelName);
-            if (val == null)
```

Comment:
How come this `if` was dropped?

Reply:
val can't be null here because it's not of a nullable type. ValueProviderResult is a struct which never can be null => therefore `if (val == null)` is always false.